CMakePresets.json

```diff
@@ -85,6 +82,7 @@
       "hidden": true,
       "description": "Disable debug mode",
       "cacheVariables": {
+        "BUILD_SHARED_LIBS":{"type": "BOOL",   "value": "OFF"},
```

Comment:
Building without shared libraries improves app load times but is incompatible with ROOT.

Reply:
I am confused by this statement.  This section of settings does not seems to mention while seemingly using the feature that this said not to work with ROOT.